server/config/config.js

```diff
@@ -0,0 +1,4 @@
+module.exports  = {
+    secretKey : "superSecrete",
+    mongoUri : "mongodb://localhost:27017/just-notes"
+}
```

Comment:
Add a line in the end

Reply:
Let's rename this file to env.js, so that later we could move the stuff inside to setup environment variables using `dotenv`

Reply:
ok

Reply:
done